src/Microsoft.DotNet.XHarness.iOS/ErrorKnowledgeBase.cs

```diff
@@ -16,17 +16,30 @@ public class ErrorKnowledgeBase : IErrorKnowledgeBase
         private static readonly Dictionary<string, (string HumanMessage, string? IssueLink)> s_testErrorMaps = new Dictionary<string, (string HumanMessage, string? IssueLink)>
         {
             ["Failed to communicate with the device"] = // Known issue but not a failure.
-                (HumanMessage: "Failed to communicate with the device. Please ensure the cable is properly connected, and try rebooting the device", IssueLink: (string?)null)
+                ("Failed to communicate with the device. Please ensure the cable is properly connected, and try rebooting the device", null),
 
+            ["the device is locked"] =
+                ("Cannot launch the application because the device is locked. Please unlock the device and try again", null),
```

Comment:
should we also grep for the `MT1031`? that one is at least documented: https://docs.microsoft.com/en-us/xamarin/ios/troubleshooting/mtouch-errors#mt1031-could-not-launch-the-app--on-the-device--because-the-device-is-locked-please-unlock-the-device-and-try-again